### DISCUSSION TITLE
Remove flow events from eve-.json?

### DISCUSSION THREAD

**Discussion by mauricefm** [Security-Onion-Solutions/securityonion#4418] [Unsupported Versions]
Hello
I'm on v2.3.50 on CentOS 7 as a standalone, installed from the ISO image.
so-status and salt-call state.highstate are both good to go. Everything 'OK' and failures 0.
How do I filter out the '"event_type":"flow"'s from the eve ... .json file? My goal is to forward them to Splunk, but the files are too large.
There is no clear place to disable inside the /opt/so/saltstack/local/pillar/global.sls or opt/so/saltstack/local/pillar/minions/<mySvr>_ standalone.sls files.  Any edits in /root/SecurityOnion/salt/suricata/defaults.yaml don't stay. (might I be editing it incorrectly...?)
Thanks in advance

**Comment by mauricefm** [comment]
Found it... 
/opt/so/saltstack/default/salt/suricata/suricata_meta.yaml

Comment out (#) the events there.

//

**Reply by mauricefm** [reply]
more specific...

To remove 'event type's, (ie, dns and flows) from /nsm/suricata/event ----- .json 
NOTE: settings will  not keep after system update (soup); backup the file first, or reaccomplish after update
(if anyone already knows how to make the changes permanent, plz tell!)

 - Comment out (#) the events here:
	/opt/so/saltstack/default/salt/suricata/suricata_meta.yaml

**#**- dns:
                version: 2

and

**#**- flow

 - run sudo so-restart suricata
 -- look for your changes in the output:
...
          ID: suriconfigsync
    Function: file.managed
        Name: /opt/so/conf/suricata/suricata.yaml
      Result: True
     Comment: File /opt/so/conf/suricata/suricata.yaml ***updated**
     Started: 13:23:12.580680
    Duration: 106.308 ms
     ***Changes**:
              ----------
              diff:
                  ---
                  +++
                  @@ -72,7 +72,6 @@
                           packethdr: 'no'
                       - http:
                           extended: 'yes'
                  ***-**    - dns:
                           version: 2
                           enabled: 'yes'
                           formats:
                  @@ -100,7 +99,6 @@
                           enabled: 'yes'
                           extended: 'yes'
                       - ssh
                  ***-**    - flow
                   - unified2-alert:
                       enabled: 'no'
                   - http-log:
 
end of output:
Summary for local
-------------
Succeeded: 17 _(changed=2)_
Failed:     0
-------------**

 - to **filter out an alert**, find (do NOT edit) the alert in:
	/opt/so/rules/nids/all.rules
 -- look for/copy the 'sid:'_**number**_ in the line
 -- run sudo so-rule disabled add **number**

 - run sudo so-rule-update (monitor the output)

 - check:
@SO suricata]# sudo so-rule disabled list
Disabled rules: **number**

**Comment by eephillip** [comment]
Doing similar
`/opt/so/saltstack/default/salt/suricata/suricata_meta.yaml`
```
suricata:
  config:
    outputs:
      - file-store:
          version: 2
          enabled: "yes"
          dir: /nsm/extracted
          xff:
            enabled: "no"
            mode: extra-data
            deployment: reverse
            header: X-Forwarded-For
      - eve-log:
          types: []
```

**Reply by Eternally8** [reply]
you can cp  /opt/so/saltstack/default/salt/suricata/defaults.yaml /opt/so/saltstack/local/salt/suricata
and you vim defaults.yaml
this can make the changes permanent
